tigerpath/majors_and_certificates/majors/POL_2018.json

```
@@ -0,0 +1,438 @@
+            "POL 378",
+            "POL 384",
+            "POL 386",
+            "POL 430",
+            "POL 431",
+            "POL 432",
+            "POL 433",
+            "POL 434",
+            "POL 479"
+          ]
+        },
+        {
+          "name": "International Relations",
+          "max_counted": 10,
+          "min_needed": null,
+          "explanation": "the IR sub-field focuses on the study of politics among nations and non-state actors in world affairs including subjects such as the causes of war, the role of international law and institutions, economic interdependence, and cooperation to advance common goals for human rights and environmental protection. POL 240, 313, 380, 381, 385, 386, 388, 389, 392, 393, 440, 441, 442, 443, 444",
+          "course_list": [
+            "POL 240",
+            "POL 313",
+            "POL 380",
+            "POL 381",
+            "POL 385",
+            "POL 386",
+            "POL 388",
+            "POL 389",
+            "POL 392",
+            "POL 393",
+            "POL 440",
+            "POL 441",
+            "POL 442",
+            "POL 443",
+            "POL 444"
+          ]
+        },
+        {
+          "name": "Methods in political science",
+          "max_counted": 10,
+          "min_needed": null,
+          "explanation": "(cannot be the primary field): These courses provide undergraduate students with analytical tools they can use to conduct rigorous social science research. POL 250, 345, 346, 347, 450, 451, 452, 453",
+          "course_list": [
+            "POL 250",
+            "POL 345",
+            "POL 346",
+            "POL 347",
+            "POL 450",
+            "POL 451",
+            "POL 452",
+            "POL 453"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Analytic Requirement",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "The department maintains a list of politics courses that have an emphasis on methodological tools for research in political science. Concentrators are required to take a course to fulfill the analytic requirement (link is external), normally no later than the first term of their junior year. The courses used to fulfill the analytic requirement cannot be used to fulfill primary-, secondary-, or third-field requirements. The analytic requirement may be satisfied by POL 341, POL 345/SOC 305, POL 346, or POL 347. We will also accept ANT 300A, ANT 301A, ECO 202, ECO 302, ECO 312, ORF 245, PHI 201, SOC 404, WWS 200, or WWS 332.",
+      "completed_by_semester": 6,
+      "course_list": [
+        "POL 341",
+        "POL 345/SOC 305",
+        "POL 346",
+        "POL 347",
+        "ANT 300A",
+        "ANT 301A",
+        "ECO 202",
+        "ECO 302",
+        "ECO 312",
+        "ORF 245",
+        "PHI 201",
+        "SOC 404",
+        "WWS 200",
+        "WWS 332"
+      ]
+    },
+    {
+      "name": "Track",
+      "max_counted": 1,
+      "min_needed": null,

```

Isn't at least one track required?
```suggestion
      "min_needed": 1,
```

---

I think they're optional. https://ua.princeton.edu/academic-units/department-department-politics#

---

I see